akeneo-design-system/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,181 @@
+import React, {FC} from 'react';
+import styled, {css} from 'styled-components';
+import {getColor, AkeneoThemedProps} from '../../theme';
+
+type PaginationProps = {
+  /**
+   * The current page number (starting à 1).
+   */
+  currentPage: number;
+
+  /**
+   * The total number of pages.
+   */
+  itemsTotal: number;
```

Comment:
```suggestion
  totalItems: number;
```
Seems more english to me but I'm maybe wrong